### PR TITLE
Add RSE form to FAB

### PR DESCRIPTION
**Pull request by shivangag**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,6 +27,11 @@
     caption: 'Contact Us'
     url: 'http://contact.robotix.in'
     target: '_blank'
+  -
```
Comment (narayanaditya95) [inline index.html]: This shouldn't be on the homepage.

File: rse.markdown [hunk 2]
```diff
@@ -3,6 +3,12 @@ layout: post
 title: Robotix Student Executive
 permalink: /rse/
 image: img/sections-background/rse.jpg
+actions:
+  -
+    text: 'F'
+    caption: 'Form'
```
Comment (narayanaditya95) [inline rse.markdown]: Caption should be `Register` or `Registration Form`.

File: index.html (hunk 1 above)
Comment (narayanaditya95) [reply index.html]: If you want a notice on the homepage, use <https://nectar.ninja>

**Comment by shivangag** [comment]
Changes made. Merging it for now.

**Pull request merged**